Samsung/WATT
# Prevent anonymous user from accessing projects list

[Issue] http://suprem.sec.samsung.net/jira/browse/TIZENWF-2614
[Problem] Anonymous user can access projects list
[Solution] Redirect to WATT home page if anonymous user attempts
           to access projects list. In addition logout anonymous
           user since WATT home page redirects to projects list
           if any user is authticated.
[Test]
    1. Open http://localhost:3000/demos/?path=mobile%2FUIComponents%2Fcomponents%2Fcontrols%2Fcheckbox.html
    2. Open http://localhost:3000/project
    3. WATT home page should be displayed.

Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>
